pkg/commands/flagsets.go

```diff
@@ -28,6 +28,8 @@ func setupLintersFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
 	fs.StringSliceP("presets", "p", nil,
 		color.GreenString(fmt.Sprintf("Enable presets (%s) of linters. Run 'golangci-lint help linters' to see "+
 			"them. This option implies option --disable-all", strings.Join(lintersdb.AllPresets(), "|"))))
+
+	fs.StringSlice("only", nil, color.GreenString("Override linters configuration section to only run the specific linter(s)")) // Flags only.
```

Comment:
Maybe we should name this parameter `--enable-only`? It would align well with `--enable` and `--enable-all`.

Reply:
I feel a bit verbose :thinking:

Reply:
+1 for `--enable-only` / `-EO`

Reply:
I was thinking about this as well in my first post, maybe clarifying the name would help. I was about to suggest `--only-linter` but didn't like it, however `--enable-only` feels pretty good. I don't think it's verbose, with just `--only` it's not clear only what.

So +1 for me as well.

Reply:
I added `--enable-only` :wink:

Reply:
Yeah let's not add a shorthand for this, there's none for `--disable-all` and `--enable-all` anyways.